### PR TITLE
motion comments in the motion detail view

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: client/src/app/site/motions/components/motion-comments/motion-comments.component.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            </button>
+            <button mat-icon-button *ngIf="isCommentEdited(section)" (click)="cancelEditing(section)">
+                <mat-icon>close</mat-icon>
+            </button>
+        </mat-action-row>
+    </mat-expansion-panel>
+</ng-template>
+
+<ng-template #desktopView>
+        <mat-card class="meta-text-block" *ngFor="let section of sections">
```
Comment (tsiegleauq) [inline client/src/app/site/motions/components/motion-comments/motion-comments.component.html]: this pretty much duplicates the "personal notes" part of motion-details, where the mobile version of this _kind of_ duplicates an entry in the motion-category and motion-comment.

We could convert this into an own component to re-use a lot of styling and functionality. Might require multi-slot-transclusion.

**Comment by FinnStutzenstein** [comment]
@tsiegleauq Ping! Additional things: the meta-text-block component and the personal note component/service

File: client/src/app/site/motions/components/motion-comments/motion-comments.component.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+     */
+    public cancelEditing(section: ViewMotionCommentSection): void {
+        delete this.commentForms[section.id];
+    }
+
+    /**
+     * Returns true, if the comment is edited.
+     * @param section The section for the comment.
+     */
+    public isCommentEdited(section: ViewMotionCommentSection): boolean {
```
Comment (tsiegleauq) [inline client/src/app/site/motions/components/motion-comments/motion-comments.component.ts]: Could use the forms `stateChanges` property

File: client/src/app/site/motions/components/personal-note/personal-note.component.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        });
+    }
+
+    /**
+     * Sets up the form.
+     */
+    public editPersonalNote(): void {
+        this.personalNoteForm.reset();
+        this.personalNoteForm.patchValue({
+            note: this.personalNote.note
```
Comment (tsiegleauq) [inline client/src/app/site/motions/components/personal-note/personal-note.component.ts]: throws:
`Cannot read property 'note' of null`

File: client/src/app/core/services/operator.service.ts [hunk 4]
```diff
@@ -132,6 +136,29 @@ export class OperatorService extends OpenSlidesComponent {
         });
     }
 
+    /**
+     * Returns true, if the operator is in at least one group or he is in the admin group.
```
[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply client/src/app/core/services/operator.service.ts]: `A rest parameter must be of an array type.` So (sadly) no..

File: client/src/app/site/motions/components/personal-note/personal-note.component.ts (hunk 3 above)
Comment (FinnStutzenstein) [reply client/src/app/site/motions/components/personal-note/personal-note.component.ts]: Should work now.

File: client/src/app/site/motions/components/motion-comments/motion-comments.component.ts [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+        });
+    }
+
+    /**
+     * Puts the comment into edit mode.
+     * @param section The section for the comment.
+     */
+    public editComment(section: ViewMotionCommentSection): void {
+        const comment = this.comments[section.id];
+        const form = this.formBuilder.group({
```
[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply client/src/app/site/motions/components/motion-comments/motion-comments.component.ts]: Yeah I know... ;)

**Comment by FinnStutzenstein** [comment]
@tsiegleauq Should work now.

**Pull request merged**